docs/guides/qiskit-addons-mpf.ipynb

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
<p>The following sentence part does not work:</p><p> </p><blockquote>are executed to obtain several measurements of the target observable&nbsp;O(t)&nbsp;need to be acquired.</blockquote><p> </p><p>Right after the equation that shows how to compute the O(t) expectation value, it would be good to generalize the coefficients to be time-dependent, and then also call out the dynamic MPFs (which are completely omitted from this PR right now).</p><p> </p><p>Need to fix this sentence as follows (correction in bold):</p><p> </p><blockquote>Each circuit should then be executed on a QPU <strong>to</strong> reconstruct the results </blockquote><p> </p><p> </p><p>In the section on determining the MPF coefficients, the parenthesis pointing out that the coefficients must be time independent has to be removed as the dynamic MPFs are now available.</p><p> </p><p>The typesetting of "largest" in the first enumeration item is broken.</p><p> </p><p>The enumeration items are inconsistent w.r.t. ending in a <code>.</code> or not. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
> The following sentence part does not work:
> 
> 
> > are executed to obtain several measurements of the target observable O(t) need to be acquired.
> 
> 
> Right after the equation that shows how to compute the O(t) expectation value, it would be good to generalize the coefficients to be time-dependent, and then also call out the dynamic MPFs (which are completely omitted from this PR right now).
> 
> 
> Need to fix this sentence as follows (correction in bold):
> 
> 
> > Each circuit should then be executed on a QPU **to** reconstruct the results
> 
> 
> 
> In the section on determining the MPF coefficients, the parenthesis pointing out that the coefficients must be time independent has to be removed as the dynamic MPFs are now available.
> 
> 
> The typesetting of "largest" in the first enumeration item is broken.
> 
> 
> The enumeration items are inconsistent w.r.t. ending in a `.` or not.
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/#comment-1857973613)_

@mrossinek thanks for the feedback, I think I've addressed all of your comments. Feel free to check and see if I've missed anything.